actions/views/drafts.ts

```diff
@@ -28,12 +30,32 @@ type Draft = {
 }
 
 export function getDrafts(teamId: string) {
-    return async (dispatch: DispatchFunc) => {
-        let drafts: ServerDraft[] = [];
-        drafts = await Client4.getUserDrafts(teamId);
-        const actions = (drafts || []).map((draft) => {
-            const {key, value} = transformServerDraft(draft);
-            return setGlobalItem(key, value);
+    const getLegacyDrafts = makeGetLegacyDrafts();
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState() as GlobalState;
+
+        let serverDrafts: Draft[] = [];
+        if (syncedDraftsAreAllowedAndEnabled(state)) {
+            serverDrafts = (await Client4.getUserDrafts(teamId) || []).map((draft) => transformServerDraft(draft));
+        }
+
+        const legacyDrafts = (getLegacyDrafts(state) || []).map((draft) => transformLegacyDraft(draft));
+        const drafts = [...serverDrafts, ...legacyDrafts];
+
+        if (drafts === null) {
+            return;
```

Comment:
please make sure to always return at least `{data: false}` so we stay consistent, because sometimes we check for `res.data` and if we have a `void` returnvalue we run into an error, because `res` is undefined.